src/redux/reducers/authReducer.js

```diff
@@ -0,0 +1,24 @@
+import { SIGNUP_ERROR, SIGNUP_SUCCESS } from '../actions/actionsTypes';
+
+const initialState = {
+  user: null,
+  error: null,
+  isLoggedIn: false,
+};
+
+const authReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SIGNUP_SUCCESS:
+      return {
+        ...state, user: action.payload, isLoggedIn: true,
+      };
+    case SIGNUP_ERROR:
+      return {
+        ...state, error: action.payload,
+      };
```

Comment:
isn't it safer to turn back user and isLoggedIn to null when there is an error

Reply:
done!!